### ISSUE TITLE
IPC Server readme example

### ISSUE THREAD

**Issue by shriphani** [paritytech/jsonrpc#272]
The example in README.md doesn't work if I include the project in Cargo.toml. It builds fine if I clone it and do `cargo build`. What is going on?

```
     let server = Server::new("/tmp/json-ipc-test.ipc", io).unwrap();
   |                  ^^^^^^^^^^^ function or associated item not found in `jsonrpc_ipc_server::Server`
```

**Comment by gnunicorn** [comment]
Nah, it's rather, the Readme wasn't updated to the latest changes. This uses a ServerBuilder pattern now. The example-code should be:
```rust
extern crate jsonrpc_ipc_server;

use jsonrpc_ipc_server::ServerBuilder;
use jsonrpc_ipc_server::jsonrpc_core::*;

fn main() {
	let mut io = IoHandler::new();
	io.add_method("say_hello", |_params| {
		Ok(Value::String("hello".into()))
	});

	let builder = ServerBuilder::new(io);
	let server = builder.start("/tmp/json-ipc-test.ipc").expect("Couldn't open socket");
	server.wait();
}
```

**Comment by tomusdrw** [comment]
Yup, thanks @gnunicorn. PR's welcome.

**Comment by ebkalderon** [comment]
Can this issue be safely closed? #400 has already been merged into `master` and supposedly resolves this.

**Comment by tomusdrw** [comment]
Indeed, thanks!
